src/sections/Footer.scss

```diff
@@ -0,0 +1,68 @@
+h1,
+h2 {
+  font-weight: 900;
+  font-size: 3.2rem;
+  text-align: center;
+  width: 100%;
+  color: white;
+}
+
+footer {
+  background-image: linear-gradient(90deg, #fdcf0c, #f89b21);
+}
+
+.socialMediaIconTitle {
+  font-size: 3.2rem;
+}
+
+.copyright {
```

Comment:
There is a margin on the copyright text that is creating a thin white line on the bottom of the page. I tried to see where it came from, but I couldn't find the source.

Viewed on Firefox on a 13' Macbook Air

Reply:
The reason is because line 20 should be margin-bottom: 0 rather than just bottom

Reply:
Thanks for pointing it out!